pipelines/controller/src/main/resources/templates/view.html

```diff
@@ -0,0 +1,150 @@
+      var resourceContainer = document.getElementById("jsoneditorResource");
+      // We want the editors variables to be global.
+      viewEditor = new JSONEditor(viewContainer, options);
+      resourceEditor = new JSONEditor(resourceContainer, options);
+    }
+  </script>
+</head>
+
+<body onload="makeEditor()">
+
+<div class="container-fluid">
+  <h1>ViewDefinition Editor</h1>
+  Here you can edit and test ViewDefinition resources. Enter your
+  ViewDefinition resource in the left editor and a sample resource on the right.
+  Then press the "Submit/Run" button to calculate the flat rows generated by
+  applying the ViewDefinition on the given resource. The JSON editors are
+  based on the <a href="https://github.com/josdejong/jsoneditor">JSON Editor repo</a>.
+
+  <div class="row m-4">
+    <div class="col text-center">
+      <button
+          data-bs-toggle="tooltip"
+          data-bs-placement="top"
+          data-bs-trigger="hover"
+          title="Use Submit to apply the ViewDefinition on the FHIR resource."
+          type="submit"
+          onclick="submitView()"
+          class="btn btn-primary">Submit/Run
+      </button>
+    </div>
+  </div>
+  <div class="row">
+    <div class="alert alert-info alert-dismissible fade show" id="viewWait" style="display:none">
+      <!-- Spinner -->
+      <div class="lds-default">
+        <div></div><div></div><div></div><div></div>
+        <div></div><div></div><div></div><div></div>
+        <div></div><div></div><div></div><div></div>
+      </div>
+      The view is being generated ...
+    </div>
+    <div class="alert alert-danger alert-dismissible fade show" id="viewError"
+         style="display:none; color:red">
+      Error in generating the view!
+      <!-- button type="button" class="btn-close" data-bs-dismiss="alert" aria-label="Close"></button -->
+    </div>
+  </div>
+  <div class="row mb-4">
+    <!-- Note each row is divided into 12 cells, so a col-2 spans two cells, ess:
+      https://getbootstrap.com/docs/5.0/layout/grid/ -->
+    <div class="col-12">
+      <div id="outputHelp" class="form-text">
+        The generated view will be displayed here.
+      </div>
+      <table class="table table-striped table-bordered" id="viewOutput"
+             aria-describedby="outputHelp">
+      </table>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col">
+      <label for="jsoneditorView" class="form-label">Input ViewDefinition</label>
+    </div>
+    <div class="col">
+      <label for="jsoneditorResource" class="form-label">Input Resource</label>
+    </div>
+  </div>
+  <div class="row">
+    <div class="col">
+      <div id="viewHelp" class="form-text">
+        Enter a ViewDefinition to be applied on the given FHIR resource on the right.
+      </div>
+      <div id="jsoneditorView" style="height: 1200px;"></div>
+      <!-- textarea class="form-control" id="inputView" aria-describedby="viewHelp"
```

Comment:
nit: I think this can be removed now, ditto for the next editor as well.

Reply:
Done.